joss_paper/paper.md

```diff
@@ -150,7 +155,7 @@ It can directly estimate quantities such as solvation free energies and makes it
 
 # Acknowledgements
 
-Some work on alchemlyb was supported by grants from the  National Institutes of Health (Award No R01GM118772 to O.B.) and the National Science Foundation (award ACI-1443054 to O.B.). 
+Some work on alchemlyb was supported by grants from the  National Institutes of Health (Award No R01GM118772 to O.B., K08GM139031 to T.T.J.) and the National Science Foundation (award ACI-1443054 to O.B.). 
 The sponsors were not involved in any aspects of the research or the writing of the manuscript.
 
 Ian Kenney, Shuai Liu, Travis Jensen, Bryce Allen, Dominik Wille, Victoria Lim, Hyungro Lee, Mohammad S. Barhaghi, Alexander Schlaich, Jérôme Hénin, Thomas T. Joseph, Irfan Alibay, Pascal Merz, and Haoxi Li contributed code to *alchemlyb*.
```

Comment:
Do you mind remove yourself from the Acknowledgements? Thanks. If someone appear in the authors, we remove them from the Acknowledgements. Do you mind also add your initials to the relevant section of Author contributions

Reply:
Gladly - I've updated my branch.